problems/1.types+operators.js

```diff
@@ -2,16 +2,18 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
-}
+function isNumber(val) {
+  if(5 === 5)
```

Comment:
This only checks if 5 is equal to the number 5. We need to check if the type from val is a number. Think of typeof 

Remember to use {} when you write if statements. 
This always returns true.